### PR TITLE
Show error when there's no storge classes

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2022-06-01 at 6 57 36 PM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/TenantResources/NamespaceSelector.tsx [hunk 1]
```diff
@@ -59,10 +59,6 @@ const NamespaceSelector = ({ formToRender }: { formToRender?: IMkEnvs }) => {
     }
   }, [debounceNamespace, namespace]);
 
-  const addNamespace = () => {
-    dispatch(openAddNSModal());
-  };
-
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/AddTenant/Steps/TenantResources/NamespaceSelector.tsx:65]: This avoids re-defining this function on every render. Please dont remove

**Review by bexsoft — CHANGES_REQUESTED** [review]
Just this, beside of that, works good

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/TenantResources/NamespaceSelector.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/AddTenant/Steps/TenantResources/NamespaceSelector.tsx]: done

**Review by prakashsvmx — APPROVED** [review]
Looks good to me. :+1:  
My Local kind cluster has standard by default for storageclass

**Comment by dvaldivia** [comment]
@prakashsvmx if you use kind, you can delete it to test this

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**